app/feature/feature-movingflow/src/main/kotlin/com/hedvig/android/feature/movingflow/data/MovingFlowQuotes.kt

```diff
@@ -0,0 +1,108 @@
+package com.hedvig.android.feature.movingflow.data
+
+import com.hedvig.android.core.uidata.UiMoney
+import com.hedvig.android.data.productVariant.android.toProductVariant
+import com.hedvig.android.data.productvariant.ProductVariant
+import com.hedvig.android.feature.movingflow.data.MovingFlowQuotes.DisplayItem
+import com.hedvig.android.feature.movingflow.data.MovingFlowQuotes.MoveHomeQuote
+import com.hedvig.android.feature.movingflow.data.MovingFlowQuotes.MoveHomeQuote.Deductible
+import com.hedvig.android.feature.movingflow.data.MovingFlowQuotes.MoveMtaQuote
+import kotlinx.datetime.LocalDate
+import kotlinx.serialization.Serializable
+import octopus.feature.movingflow.fragment.MoveIntentQuotesFragment
+import octopus.feature.movingflow.fragment.MoveQuoteDisplayItemFragment
+
+@Serializable
+internal data class MovingFlowQuotes(
+  val homeQuotes: List<MoveHomeQuote>,
+  val mtaQuotes: List<MoveMtaQuote>,
+) {
+  interface Quote {
+    val premium: UiMoney
+    val exposureName: String
+    val productVariant: ProductVariant
+    val startDate: LocalDate
+    val displayItems: List<DisplayItem>
+  }
+
+  @Serializable
+  data class MoveHomeQuote(
+    val id: String,
+    override val premium: UiMoney,
+    override val startDate: LocalDate,
+    override val displayItems: List<DisplayItem>,
+    override val exposureName: String,
+    override val productVariant: ProductVariant,
+    val tierName: String,
+    val tierLevel: Int,
+    val tierDescription: String?,
```

Comment:
would it maybe be convenient to put everything tier-related in one Tier entity (incl. productVariant.displayNameTier)?

Reply:
I ended up grabbing the displayNameTier by doing this
```
val tierDisplayName = productVariant.displayTierName ?: tierName
```
Which I looked at Ugglan and they do the exact same thing.

So I am not sure, would the tier also include the ProductVariant in there? Since it needs it to grab the real displayable name.
But then, the ProductVariant exists in both type of quotes here, so extracting one Tier data class out of this would be tricky.

I think I won't do anything in particular for this yet, and as we work more on this if we find some good way to make this even more cohesive I'd be glad to do it